examples/lode_linear/environement.yml

```diff
@@ -0,0 +1,15 @@
+name: lode_linear
+dependencies:
+  - python=3.11
+  - pip
+  - pip:
+    # sphinx-gallery and dependencies
+    - sphinx
+    - pillow
+    - sphinx-gallery
+    - matplotlib
```

Comment:
not really a sphinx-gallery example. We could have example that do not have to plot anything.

Reply:
everything is a sphinx-gallery example, since that's what we use to transform the python script to rst